src/ec/suite_b/ops/ops.rs

```diff
@@ -215,11 +210,9 @@ impl CommonOps {
 
     pub fn elem_verify_is_not_zero(&self, a: &ElemUnreduced)
                                    -> Result<(), error::Unspecified> {
-        match unsafe {
-            GFp_constant_time_limbs_are_zero(a.limbs.as_ptr(), self.num_limbs)
-        } {
-            0 => Ok(()),
-            _ => Err(error::Unspecified),
+        match limbs_are_zero_constant_time(&a.limbs[..self.num_limbs]) {
+            false => Ok(()),
+            true => Err(error::Unspecified),
```

Comment:
Ok. I thought the indirection was worth it to remove an unneeded use of `unsafe` and keep the unsafe implementation private to `limb.rs`